GDAXSharp/WebSocket/Types/ResponseType.cs

```diff
@@ -15,6 +13,8 @@ public enum ResponseType
         Open,
         Done,
         Match,
+        [EnumMember(Value = "last_match")]
```

Comment:
@chrisw000 @dougdellolio  Is this the best way for deserializing `last_match` to `LastMatch` ?

Reply:
@sotam - yep